src/haddock/modules/analysis/caprieval/defaults.cfg

```diff
@@ -3,9 +3,10 @@ irmsd = true
 fnat = true
 lrmsd = true
 ilrmsd = true
-irmsd_cutoff = 10.0
+irmsd_cutoff = 5.0
```

Comment:
Hum - isn't CAPRI using all interface residue making contacts within 10A?  At least for protein-protein